From b6e1945383b340510f2cb103f0482b32bd1b1cf4 Mon Sep 17 00:00:00 2001
From: Jeremie Graulle <jeremiex.graulle@intel.com>
Date: Tue, 2 Apr 2013 12:57:58 +0200
Subject: [PORT FROM MAIN] libmedia_helper: System include for host

BZ: 93434

Compilation error for host NULL is not define.

Add the system include stdlib.h in sources files.

Change-Id: Ic330be220aa36525c75b2b2f268bfd8c2c1874b8
Orig-Change-Id: I67c0d915722976d814bf879b3d05c1f91aee97fc
Signed-off-by: Jeremie Graulle <jeremiex.graulle@intel.com>
---
 include/media/AudioBufferProvider.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/media/AudioBufferProvider.h b/include/media/AudioBufferProvider.h
index 43e4de7..218c010 100644
--- a/include/media/AudioBufferProvider.h
+++ b/include/media/AudioBufferProvider.h
@@ -18,6 +18,7 @@
 #define ANDROID_AUDIO_BUFFER_PROVIDER_H
 
 #include <utils/Errors.h>
+#include <stdlib.h>
 
 namespace android {
 // ----------------------------------------------------------------------------
-- 
1.7.4.1

